<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeAutowidget</class>
 <widget class="QWidget" name="ModeAutowidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VL_menu_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="VL_menuTransaction_2">
      <item>
       <widget class="QLabel" name="pairelabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Paire devises</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL_infoDate"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="base">
          <property name="text">
           <string>Devise Base</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_montantBase">
          <property name="text">
           <string>MontantB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="contrepartie">
          <property name="text">
           <string>Contre partie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_montantContrepartie">
          <property name="text">
           <string>MontantC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_broker">
          <property name="text">
           <string>Broker</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Interet Broker</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/TradingSimulator/evolutionCours/icons/step.png</normaloff>:/TradingSimulator/evolutionCours/icons/step.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_strategie">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Strategie</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/TradingSimulator/evolutionCours/icons/timer.jpg</normaloff>:/TradingSimulator/evolutionCours/icons/timer.jpg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_timer">
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Open Price</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="openPrice">
          <property name="text">
           <string>00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL_achatvente_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
